README.md

```diff
@@ -110,6 +110,23 @@ depends: [
 ```
 > ℹ&#xFE0F; _[What is DDS?](https://dds.pizza/)_
 
+### Tipi.build
+
+
+`tomlplusplus` can be easily used in [tipi.build](https://tipi.build) projects by adding the following entry to your `.tipi/deps`:
+
+```json
+{
+    "marzer/tomlplusplus": { }
+}
+```
+
+> Try this by running the following command in `/examples/tipi-example` (change the `<target>` name appropriately to `linux` or `macos` or `windows`):
+>
+> ```bash
+> tipi . -t linux # or -t windows / -t macos
+> ```
+
```

Comment:
You should probably also add a section to `docs/pages/main_page.dox` - that's the front page of the [HTML docs](https://marzer.github.io/tomlplusplus/)

Reply:
Oh, also, the "Try this" section isn't correct anymore without the dedicated `examples/tipi-example`.

Reply:
Oh, also, the "Try this" section isn't correct anymore without the dedicated examples/tipi-example.